code/datums/uplink_items/uplink_traitor.dm

```diff
@@ -572,14 +572,14 @@
 /obj/structure/closet/crate/surplus
 	name = "Surplus crate"
 	var/cost
-	crate_value // This is a redeclare but we ignore for now
 
 /datum/uplink_item/bundles_TC/surplus_crate/spawn_item(turf/loc, obj/item/uplink/U)
-	var/obj/structure/closet/crate/surplus/surplus = new(loc, U)
+	var/obj/structure/closet/crate/surplus/surplus = new(loc, U, crate_value)
 	surplus.cost = cost
-	surplus.crate_value = crate_value
+	U.uses -= cost
+	U.used_TC = cost
```

Comment:
Can you make this use . = ..() instead, and cancel the spawning of the item in the parent proc?

Reply:
Guh, didn't know parent did that because surplus code did this normally, will change